tests/unit/CMakeLists.txt

```diff
@@ -33,7 +33,6 @@ set(COMMON_SOURCES
 
 set(COMMON_LIBS
     openthread-ftd
-    openthread-ncp-ftd
```

Comment:
why does `openthread-ncp-ftd` include simulation platform?

Reply:
I think so. I haven't checked the code. But I checked the `build.ninja` and I'm sure it's this one that causes `simulation` linked together.

Reply:
https://github.com/openthread/openthread/blob/master/src/ncp/CMakeLists.txt#L112

`openthread-ncp-ftd` added `OT_PLATFORM_LIB` into its link targets.